spec.html

```diff
@@ -38348,12 +38342,12 @@ <h1>Promise.race ( _iterable_ )</h1>
           <p>If the _iterable_ argument is empty or if none of the promises in _iterable_ ever settle then the pending promise returned by this method will never be settled.</p>
         </emu-note>
         <emu-note>
-          <p>The `race` function expects its *this* value to be a constructor function that supports the parameter conventions of the Promise constructor. It also expects that its *this* value provides a `resolve` method.</p>
+          <p>The `anySettled` function expects its *this* value to be a constructor function that supports the parameter conventions of the Promise constructor. It also expects that its *this* value provides a `resolve` method.</p>
         </emu-note>
 
-        <emu-clause id="sec-performpromiserace" aoid="PerformPromiseRace">
-          <h1>PerformPromiseRace ( _iteratorRecord_, _constructor_, _resultCapability_, _promiseResolve_ )</h1>
-          <p>The abstract operation PerformPromiseRace takes arguments _iteratorRecord_, _constructor_, _resultCapability_ (a PromiseCapability Record), and _promiseResolve_. It performs the following steps when called:</p>
+        <emu-clause id="sec-performpromiseanysettled" aoid="PerformPromiseAnySettled">
```

Comment:
This should include the `[oldids]` attribute:
```suggestion
        <emu-clause id="sec-performpromiseanysettled" aoid="PerformPromiseAnySettled" oldids="sec-performpromiserace">
```